packages/plugin-e2e/.env.example

```diff
@@ -0,0 +1 @@
+GOOGLE_JWT_FILE=
```

Comment:
secret added to the repo. can also be found in plugin-provisioning repo if you want to try this locally (or provide your own ofc).